scala-package/mxnet-demo/scala-demo/Makefile

```diff
@@ -17,7 +17,7 @@
 
 SCALA_VERSION_PROFILE := 2.11
 SCALA_VERSION := 2.11.8
-MXNET_VERSION := 1.2.0
+MXNET_VERSION := 1.3.0
```

Comment:
`1.3.1-SNAPSHOT`? Is this necessary all the time? can this not pick latest from Maven?

Reply:
Here is just a migration. As I have learnt from here: https://stackoverflow.com/questions/30571/how-do-i-tell-maven-to-use-the-latest-version-of-a-dependency Maven has dropped the feature to get the `LATEST` so the user has to manually choose the version. The alternative way could be using plugins to target the latest version.